js/plugins/tabs.html

```diff
@@ -276,7 +276,7 @@
         <button class="tab-delete btn"><i class="icon-trash"></i> Remove</button>';
 
 
-            var id = 'bs_tab' + e.timeStamp;
+            var id = 'bs_tab' + guid();
```

Comment:
Suggestion to add an underscore after 'bs_tab' for the id. So, `var id = 'bs_tab_' + guid();`